python_modules/dagster/dagster/_core/definitions/repository_definition.py

```diff
@@ -1366,12 +1366,14 @@ def get_implicit_global_asset_job_def(self) -> JobDefinition:
 
         return self.get_job(ASSET_BASE_JOB_PREFIX)
 
-    def get_base_asset_job_names(self) -> Sequence[str]:
+    def get_implicit_asset_job_def_names(self) -> Sequence[str]:
```

Comment:
Do we need "def" in this name? I think we refer to "job names" in a bunch of other places, and IMO it's clear enough what that means.

Reply:
All the other method names refer to defs so this feels inconsistent. Seems like we need to come up with consistent guidance around this. The codebase is all over the place on this topic.

Reply:
My reading of the larger codebase is that we've settled on job_name. I.e. I see job_name in a lot of places and not much job_def_name.  E.g. DagsterRun has a job_name, ScheduleDefinition/SensorDefinition/RunRequest/JobSelector all accept a job_name, the CLI talks about job_names.

I don't think I'd find it weird to come across both `get_job_defs` and `get_job_names` in `Definitions`.